### PR TITLE
Add support for questions not counting towards an abstract's score

### PR DESCRIPTION
**Pull request by ThiefMaster**

### CODE REVIEW

File: indico/htdocs/js/indico/widgets/multiple_items_widget.js [hunk 1]
```diff
@@ -187,46 +191,70 @@
...
-                    sel.append($('<option>')); // Default empty option
-                    for (var choiceID in options.columnChoices[col.id]) {
-                        sel.append($('<option>', {
-                            value: choiceID,
-                            text: options.columnChoices[col.id][choiceID],
-                            selected: item ? item[col.id] === choiceID : false
-                        }));
+                    return {text: options.columnChoices[col.id][item[col.id]]};
+                } else if (col.type === 'checkbox') {
+                    // return {text: item[col.id] ? '✓' : ''};
```
Comment (nop33) [inline indico/htdocs/js/indico/widgets/multiple_items_widget.js]: forgotten?

File: indico/htdocs/js/indico/widgets/multiple_items_widget.js [hunk 2]
```diff
@@ -187,46 +191,70 @@
...
+            } else if (col.type === 'checkbox') {
+                return {
+                    html: $('<input>', {
+                        type: 'checkbox',
+                        class: 'js-table-input',
+                        value: '1',
+                        checked: item ? item[col.id] : false
+                    }),
+                    css: {
+                        width: '1%'
```
Comment (nop33) [inline indico/htdocs/js/indico/widgets/multiple_items_widget.js]: When the dict has only 1 element, why not having it inline (like you do in line `222`)?

File: indico/htdocs/js/indico/widgets/multiple_items_widget.js (hunk 1 above)
Comment (ThiefMaster) [inline indico/htdocs/js/indico/widgets/multiple_items_widget.js]: yes

File: indico/htdocs/js/indico/widgets/multiple_items_widget.js (hunk 2 above)
Comment (ThiefMaster) [inline indico/htdocs/js/indico/widgets/multiple_items_widget.js]: consistency within the statement

**Pull request merged**